registration-system/admin/pages_mail.php

```diff
@@ -37,10 +37,13 @@ public function getText() {
         foreach ($this->environment->oconfig['essen'] as $key => $typ) {
             $essen .= '<option value="' . $key . '">' . $typ . '</option>';
         }
-        $maxtage = $this->fahrt->getLenTage();
+        $fahrt_bereich = $this->environment->database->select('fahrten', ['von', 'bis'], ['fahrt_id' => $this->fahrt->getID()]);
+        $von = strtotime($fahrt_bereich[0]['von']);
+        $bis = strtotime($fahrt_bereich[0]['bis']);
+
         $tage = '';
-        for ($cnt = $maxtage; $cnt >= 0; $cnt--)
-            $tage .= '<option value="' . $cnt . '">' . $cnt . '</option>';
+        for ($tag = $von; $tag < $bis; $tag += 24*60*60)
+            $tage .= '<option value="' . date('Y-m-d', $tag) . '">' . date('d.m.Y', $tag) . '</option>';
```

Comment:
warum zwei verschiedene datumsformate? ist das in der DB so? mir ist so, als wäre da irgendwo im environment eine funktion für datumsangaben. wäre wünschenswert die zu nutzen!

Reply:
datumsformate sind so in der db - das rechte ist 'menschenleserlich'